spec/factories/clients.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :client do
+    email { Faker::Internet.email }
```

Comment:
@DmitrS did you try to use `create` instead of `build`, or not to add `encrypted_password` to fabric?